### PR TITLE
Add docs for OSX install

### PR DESCRIPTION
**Pull request by dflynn15**
Adds documentation on how to install and run `guetzli` on OSX.

### CODE REVIEW

**Comment by dflynn15** [comment]
I signed it!

**Comment by btnwtn** [comment]
@dflynn15 Would you mind changing OSX to OS X?

**Comment by dracos** [comment]
You could also include MacPorts with:

2.  Install [MacPorts](https://www.macports.org/)
3.  Install `libpng` and `gflags` using port: `sudo port install libpng gflags`.
4.  Run `CFLAGS='-I/opt/local/include' LDFLAGS='-L/opt/local/lib' make` and expect the binary to be created in `bin/Release/guetzli`.

Hopefully straightforward to get guetzli added to both brew and port directly too, given that's all it needs in either.

**Comment by fgilio** [comment]
@btnwtn @dflynn15 actually, it's **macOS** nowadays

**Comment by dflynn15** [comment]
@btnwtn @fgilio @dracos I went ahead and updated with the suggestions. Take a look and let me know if anything else needs to be changed!

File: README.md [hunk 1]
```diff
@@ -35,6 +35,20 @@ sequential (nonprogressive) JPEGs due to faster decompression speeds they offer.
...
+    downloading an
+    [archive](https://github.com/google/guetzli/archive/master.zip) and
+    unpacking it.
+2.  Install [Homebrew](https://brew.sh/) or [MacPorts](https://www.macports.org/)
+3.  Install `libpng` and `gflags` 
+    *   Using Homebrew: `brew install libpng gflags`.
+    *   Using MacPorts: `port install libpng gflags` (You may need to use `sudo`).
+4.  Run the following command to build the binary in `bin/Release/guetzli`.
+    *   If you installed using Homebrew simply use `make`
+    *   If you installed using Macports use `CFLAGS='-I/opt/local/include' LDFLAGS='-L/opt/local/lib'`
```
Comment (dracos) [inline README.md]: Nice, thanks! One tiny thing, this line needs a `make` at the end :)

**Comment by fkistner** [comment]
@dflynn15 Note, that guetzli has been included in Homebrew, so a simple `brew install guetzli`will work, too.

**Comment by dflynn15** [comment]
@fkistner updated 👍

**Comment by robryk** [comment]
Thank you very much!

**Pull request merged**